src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -147,11 +174,25 @@ impl KvmVcpu {
         };
         get_all_registers(&self.fd, &mut state.regs).map_err(KvmVcpuError::SaveState)?;
         state.mpidr = get_mpidr(&self.fd).map_err(KvmVcpuError::SaveState)?;
+        state.kvi = self.kvi;
         Ok(state)
     }
 
     /// Use provided state to populate KVM internal state.
-    pub fn restore_state(&self, state: &VcpuState) -> Result<(), KvmVcpuError> {
+    pub fn restore_state(
```

Comment:
so at this point, aarch64 and x86 signatures of this method diverge. Why would it be not a good idea to keep them in line (ie make a similar change in x86)?

Reply:
I don't think this is a good idea. There will be no use for additional parameters in x86_64 method.

Reply:
But we could avoid conditional compilation. Do you think it is less valuable?

Reply:
I don't think it is worth it. aarch64 already have `vcpu_init` as an additional method. It is not worth trying to unite interfaces now. We can think about it when we refactor `builder.rs` in some theoretical future.

Reply:
ok, fair enough.